docs/src/pages/reference/configuration/output.md

```diff
@@ -1720,3 +1720,21 @@ module.exports = {
   },
 };
 ```
+
+### optionsParamRequired
```

Comment:
I thought it would be easier to understand if you specify what option it is.

```suggestion
### axiosOptionsParamRequired
```

Reply:
I wouldn't call it only for `axios...` since it affects renderers other than axios, like `swr`

Reply:
What also affects `swr` is the instance of `axios` as `fetcher`, right?
Therefore, I think there is no problem in explicitly specifying `Axios` here, but what do you think?

Reply:
It looks like the `swr`, `query`, and `axios` modules would all share this change (I'm honeslt not familiar enough with `query` and `swr` to know if they should just take the `axios` prefix so I'll defer to your judgement on that one)

Reply:
OK, I'll think about it for a moment.

Reply:
Actually, I suggest adding a `fetch API` client, so I was concerned about the balance with that.

https://github.com/anymaniax/orval/pull/1353/files#diff-bb3fc1dad2ae5a09c4f6c065a987d0d6719e469a95aaad147885938e1bbdcddeR60

For now, we can coexist, and if necessary, we can separate the options in the future, so i agree to accept this option.

Reply:
@soartec-lab makes total sense; once the `fetch` client drops, I'm happy to integrate support for the same setup introduced here as well